lib/nerves_hub_link/health_check.ex

```diff
@@ -0,0 +1,27 @@
+defmodule NervesHubLink.HealthCheck do
```

Comment:
whats the purpose of this file?

Reply:
Will move the doc of the config stuff. Didn't want to remove them until I had put them somewhere.